x-pack/performance/journeys/ecommerce_dashboard.ts

```diff
@@ -7,50 +7,20 @@
 
 import { Journey } from '@kbn/journeys';
 import { subj } from '@kbn/test-subj-selector';
-
-import { ToastsService } from '../services/toasts';
-import { waitForVisualizations } from '../utils';
+import { waitForChrome, waitForVisualizations } from '../utils';
 
 export const journey = new Journey({
-  extendContext: ({ page, log }) => ({
-    toasts: new ToastsService(log, page),
-  }),
+  esArchives: ['x-pack/performance/es_archives/sample_data_ecommerce'],
+  kbnArchives: ['x-pack/performance/kbn_archives/ecommerce_no_map_dashboard'],
 })
-  .step('Go to Sample Data Page', async ({ page, kbnUrl }) => {
-    await page.goto(kbnUrl.get(`/app/home#/tutorial_directory/sampleData`));
-
-    await page.waitForSelector(subj('showSampleDataButton'));
-  })
 
-  .step('Open Sample Data pane', async ({ page }) => {
-    // open the "other sample data sets" section
-    await page.click(subj('showSampleDataButton'));
-    // wait for the logs card to be visible
-    await page.waitForSelector(subj('sampleDataSetCardecommerce'));
-  })
-
-  .step('Remove Ecommerce Sample Data if installed', async ({ page, log, toasts }) => {
-    if (!(await page.$(subj('removeSampleDataSetecommerce')))) {
-      log.info('Ecommerce data does not need to be removed');
-      return;
-    }
-
-    // click the "remove" button
-    await page.click(subj('removeSampleDataSetecommerce'));
-    // wait for the toast acknowledging uninstallation
-    await toasts.waitForAndClear('uninstalled');
-  })
-
-  .step('Install Ecommerce Sample Data', async ({ page, toasts }) => {
-    // click the "add data" button
-    await page.click(subj('addSampleDataSetecommerce'));
-    // wait for the toast acknowledging installation
-    await toasts.waitForAndClear('installed');
+  .step('Go to Dashboards Page', async ({ page, kbnUrl }) => {
+    await page.goto(kbnUrl.get(`/app/dashboards`));
+    await page.waitForSelector('#dashboardListingHeading');
   })
 
   .step('Go to Ecommerce Dashboard', async ({ page }) => {
-    await page.click(subj('launchSampleDataSetecommerce'));
-    await page.click(subj('viewSampleDataSetecommerce-dashboard'));
-
+    await page.click(subj('dashboardListingTitleLink-[eCommerce]-Revenue-Dashboard'));
+    await waitForChrome(page);
```

Comment:
You don't really need to wait for chrome if you're navigating to the dashboard.
It's only useful if you're using goto directly like you did before

Reply:
I will remove it then.